assets/javascripts/discourse/templates/components/tc-text.hbs

```diff
@@ -1,20 +1,15 @@
-<div class="tc-text">
-  {{#if isCollapsible}}
-    {{tc-message-collapser cooked=cooked uploads=uploads}}
-  {{else}}
-    {{html-safe cooked}}
-  {{/if}}
-  {{#if uploads.length}}
-    <div class="tc-uploads">
-      {{#each uploads as |upload|}}
-        {{chat-upload upload=upload}}
-      {{/each}}
-    </div>
-  {{/if}}
-  {{#if edited}}
-    <div class="tc-message-edited">
-      {{i18n "chat.edited"}}
-    </div>
-  {{/if}}
-  {{yield}}
-</div>
+{{#if isYoutubeCollapsible}}
+  {{tc-message-collapser cooked=cooked}}
+{{else if isImageCollapsible}}
+  {{tc-message-collapser cooked=cooked uploads=uploads}}
```

Comment:
one part of me thinks it's strange that we need to have knowledge of uploads in tc-message-collapser

Reply:
I did think hard about it, but if we think about responsibilities, I don't think I want `tc-text` to have knowledge on how to define collapsed titles or text. Even putting `isYoutubeCollapsible` is bleeding collapsing knowledge to `tc-text` already.

Ideally what I wanted is another component 😅 `text -> collapsible -> youtube-collapsible OR image-collapsible` but I decided against premature optimisation